pkg/builder/group/group_builder.go

```diff
@@ -91,7 +91,7 @@ func (groupBuilder *AbstractBuilder) Build() (*proto.GroupDTO, error) {
 	groupDTO.IsConsistentResizing = &groupBuilder.consistentResize
 
 	if groupBuilder.ec.Count() > 0 {
-		glog.Errorf("GroupBuilder Error %s : %s\n", groupBuilder.groupId, groupBuilder.ec.Error())
+		glog.Errorf("%s : %s", groupBuilder.groupId, groupBuilder.ec.Error())
```

Comment:
I would add a specific error string that says what the error is.

Reply:
The specific error is already captured in `groupBuilder.ec.Error()`. I removed the redundant string. I have a test case that demonstrates the errors being printed out:
```
E0628 17:34:05.687834   36742 group_builder.go:94] cluster1 : GroupBuilder errors: Error 0: empty member list
Cluster error failed to build group: cluster1: GroupBuilder errors: Error 0: empty member list
```